test/FilterPluginManagerTest.php

```diff
@@ -33,6 +34,7 @@ public function testFilterSuccessfullyRetrieved(): void
     public function testRegisteringInvalidFilterRaisesException(): void
     {
         $this->expectException($this->getInvalidServiceException());
+        /** @psalm-suppress InvalidArgument */
         $this->filters->setService('test', $this);
```

Comment:
I would expect the exception right before calling `get()` - as it is written here, it is a bit unclear where it may crash